.changeset/cyan-garlics-act.md

```diff
@@ -0,0 +1,6 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/types': patch
+---
+
+Disable fields for SAML only if there is an active connection
```

Comment:
Since this is supposed to be read by our customers, shall we reuse part of the PR description? 

```suggestion
Password, first name & last name will only be disabled if there are active SAML connections.
```

Reply:
Sure